src/utils/constants.js

```diff
@@ -39,9 +47,16 @@ export const initialCards = [
 /*-----------------------------------------------------------------*/
 /*                            Elements                             */
  /*-----------------------------------------------------------------*/
+
+export const confirmDeleteBtn = document.querySelector("#card-delete-btn");
 export const buttonEdit = document.querySelector('#js-edit-button');
 export const buttonAdd = document.querySelector('#js-add-button');
-export const closeButtons = document.querySelectorAll('.modal__close');
+export const changeAvatarbtn = document.querySelector('.profile__avatar-edit-btn');
+export const savingBtn = document.querySelector('.modal__save-button');
```

Comment:
You've found just a random button here


There are 3 modals with different buttons and forms in the project. You need to **name** variables and functions the way all could understand what they are.


And you need to search each button inside the corresponding modal